docs/content/en/docs/soarca-extensions/native-capabilities.md

```diff
@@ -33,4 +33,8 @@ CACAO documentation: [SSH Command](https://docs.oasis-open.org/cacao/security-pl
 
 The PowerShell capability allows executing commands on systems running an WinRM server.
 
-CACAO documentation: [PowerShell Command](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256499)
\ No newline at end of file
+CACAO documentation: [PowerShell Command](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256499)
+
+## Caldera capability
+
+This capability allows for interoperability between SOARCA and [Caldera](https://caldera.mitre.org/).
```

Comment:
This is copied over to the documentation PR #11 and can be removed here.

Reply:
check